vernac/vernacentries.ml

```diff
@@ -1937,7 +1937,7 @@ let vernac_search ~pstate ~atts s gopt r =
     match gopt with | None ->
       (* 1st goal by default if it exists, otherwise no goal at all *)
       (try get_goal_or_global_context ~pstate 1
-       with _ -> let env = Global.env () in (Evd.from_env env, env))
+       with e when CErrors.noncritical e -> let env = Global.env () in (Evd.from_env env, env))
```

Comment:
This `try` is actually dead code since 7efaf86882488034e6545505e1eda7d6e1a6ce14 started using `_or_global_context`

Reply:
So, you mean just keeping `get_goal_or_global_context ~pstate 1`? OK, then.